ibis/backends/base/sql/alchemy/translator.py

```diff
@@ -40,12 +43,41 @@ class AlchemyExprTranslator(ExprTranslator):
 
     context_class = AlchemyContext
 
+    _bool_aggs_need_cast_to_int32 = True
+    _boolean_cast_ops = ops.Sum, ops.Mean, ops.Min, ops.Max
```

Comment:
Do we really need to configure these ops per backend? If not could we use them directly in the isinstance check?

Reply:
Some backends don't require it, though the cast is likely okay for those too.

Reply:
Ah, and this is handled by `_bool_aggs_need_cast_to_int32` anyway, so yeah I'll inline them.

Reply:
Done.